.github/workflows/swift.yml

```diff
@@ -33,6 +33,10 @@ on:
       - rc/*
       - codeql-cli-*
 
+permissions:
+  contents: read
+  security-events: read
```

Comment:
?

```suggestion
```

Reply:
Maybe?

This job consistently dies:
https://github.com/check-spelling-sandbox/codeql/actions/runs/7820458953/job/21335237762

We'll see what things look like when I update this PR.

But, I'd expect it to consistently die.